tests/test_camera.py

```diff
@@ -140,9 +140,9 @@ async def test_get_image(self, camera: MockCamera, service: CameraRPCService, im
 
             # Test known mime type
             request = GetImageRequest(name="camera", mime_type=CameraMimeType.PNG)
-            response: GetImageResponse = await client.GetImage(request, timeout=18.2)
+            response: GetImageResponse = await client.GetImage(request, timeout=18.1)
```

Comment:
grpclib [rounds timeouts down to the nearest second](https://github.com/vmagamedov/grpclib/blob/b841b4e861c4ac82c7e1f3bfa0aea03dd0b9ba18/grpclib/metadata.py#L49) when serializing them over the wire. 18.2 isn't loosely equal to 18 within a tolerance of 1%, but 18.1 is! 🙃